Code/Controllers/ViewController.swift

```diff
@@ -219,7 +226,7 @@ class ViewController: UIViewController, PFLogInViewControllerDelegate, PFSignUpV
                 if error == nil {
                     let identityToken = object as! String
                     self.layerClient.authenticateWithIdentityToken(identityToken) { authenticatedUserID, error in
-                        if (!authenticatedUserID.isEmpty) {
+                        if (authenticatedUserID!.isEmpty) {
```

Comment:
Hi @jessearmand, Thanks for the PR! :-)
I think here the logic is inverted? We are supposed to check that the authenticatedUserID is not empty, then it is considered successful?